scss/base/_variables.scss

```diff
@@ -384,4 +384,7 @@ $slider-volume-thumb-diameter-small: $slider-volume-thumb-diameter * $xsmall-mul
 //bitrate
 $bitrate-horizontal-height: 100px !default;
 
-$font-path: "assets/fonts";
\ No newline at end of file
+$font-path: "assets/fonts";
+
+/gradients
```

Comment:
I feel being mildly condescending is not awesome. I could always create a million tickets and have them prioritized so you can do them @rchalooyala  ; )